RELEASE.md

```diff
@@ -0,0 +1,51 @@
+# Release process
+
+## To release a new version of **qtsass** on PyPI:
+
+* Ensure you have the latest version from upstream and update your fork
+
+      git pull upstream master
+      git push origin master
+
+* Clean the repo
+
+      git clean -xfdi
+
+* Update CHANGELOG.md
+
+* Update version in `__init__.py` (set release version, remove 'dev0')
+
+* Commit changes
+
+      git add .
+      git commit -m "Set release version"
```

Comment:
Please change this to

    git commit -m "Release X.X.X"

where `X.X.X` is the released version.